EnableConnectAsyncTests.py

```diff
@@ -0,0 +1,15 @@
+import os
```

Comment:
Why do we use python script to rewrite the tests? one has to have python installed. Cannot we use for example environment variable and keep everything in node js?

we can wrap connection manual tests in side if like that:
```
if (process.env.RUN_MANUAL_TESTS_ONLY === 'true') {
  describe.only('Run manual tests', () => {
     describe('Connection test - external browser'", "describe('Connection test - external browser', () => ...)
     // and other tests here without skip
  })
}
```

and then in package json:
```
"test:manual": "RUN_MANUAL_TESTS_ONLY=true npm run test:single -- test/integration/testConnection.js"
```

Reply:
Updated. This looks better than generate a new file using python.
Although RUN_MANUAL_TESTS_ONLY cannot be set in package.json like that. 
So I made a change to readme to set this variable before test.